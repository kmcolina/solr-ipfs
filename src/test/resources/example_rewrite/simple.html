<html>
<head>
    <title>Some page</title>
    <link rel="stylesheet" type="text/css" href="styles/index_o1.css" />
    <script type="text/javascript" src="somescript_o90.js"></script>
</head>
<body background="bodyback_o60.jpg">
<h1>A very simple page</h1>

<p>An <a href="http://example.com">absolute link</a> leading to example.com</p>
<p>A <a href="subfolder/">relative link</a> leading to <tt>subfolder</tt>subfolder</tt></p>
<p>A <a href="/otherpage.html">root-based link</a> leading to a parallel page <tt>otherpage.html</tt>.</p>

<p>An absolute referenced image: <img src="http://example.com/image_o2.jpg"/></p>
<p>A relatively referenced image: <img src="subfolder/image_o3.jpg"/></p>
<p>A root referenced image (same as the absolute one): <img src="/image_o2.jpg"/></p>

<embed src="embedded_pic_o50.jpg" width="300" height="200"/>

<audio controls>
  <source src="horse_o60.ogg" type="audio/ogg">
  <source src="horse_o61.mp3" type="audio/mpeg">
  Your browser does not support the audio element.
</audio>

<table background="table_back_o100.jpg">
    <tr><td background="td_back_o101.png">Single cell</td></tr>
</table>
</body>
</html>
