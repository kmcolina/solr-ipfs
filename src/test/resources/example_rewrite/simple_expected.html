<html>
 <head>
  <title>Some page</title>
  <link rel="stylesheet" type="text/css" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=1">
  <script type="text/javascript" src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=90"></script>
 </head>
 <body background="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=60">
  <h1>A very simple page</h1>
  <p>An <a href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com">absolute link</a> leading to example.com</p>
  <p>A <a href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/somefolder/subfolder/">relative link</a> leading to <tt>subfolder</tt>subfolder</p>
  <p>A <a href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/otherpage.html">root-based link</a> leading to a parallel page <tt>otherpage.html</tt>.</p>
  <p>An absolute referenced image: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2"></p>
  <p>A relatively referenced image: <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=3"></p>
  <p>A root referenced image (same as the absolute one): <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=2"></p>
  <embed src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=50" width="300" height="200">
  <audio controls>
   <source src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=60" type="audio/ogg">
   <source src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=61" type="audio/mpeg"> Your browser does not support the audio element.
  </audio>
  <table background="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=100">
   <tbody>
    <tr>
     <td background="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=101">Single cell</td>
    </tr>
   </tbody>
  </table>
 </body>
</html>